containers/unit-tests/setup.sh

```diff
@@ -37,7 +37,7 @@ dependencies="\
     libssh-dev \
     libsystemd-dev \
     npm \
-    nodejs:amd64 \
+    nodejs \
```

Comment:
Curious that this works now -- not sure about the background, I guess node wasn't supported on 32 bit in at least Debian 11?